tests/tests_twobody/test_states.py

```diff
@@ -35,3 +38,42 @@ def test_rv_state_has_rv_given_in_constructor():
     ss = RVState(Sun, r, v, None)
     assert (ss.r == r).all()
     assert (ss.v == v).all()
+
+
+def test_mean_motion():
+    # From Vallado Example 1-1.
+    attractor = Earth
+    period = 86164.090518 * u.s
+    a = 42164.1696 * u.km
+    # Unused variables.
+    _ecc = 0 * u.one
+    _inc = 1.85 * u.deg
+    _raan = 50 * u.deg
+    _argp = 200 * u.deg
+    _nu = 20 * u.deg
+
+    ss = ClassicalState(
+        attractor, a * (1 - _ecc ** 2), _ecc, _inc, _raan, _argp, _nu, None
+    )
+
+    expected_mean_motion = (2 * np.pi / period) * u.rad
+    n = ss.n
+
+    assert_quantity_allclose(n, expected_mean_motion)
+
+
+def test_classical_to_equinoctial_raises_singularity_error_if_inc_180_degrees():
```

Comment:
Let us explicitly say the elements being tested are `modified_equinoctial` and not just `equinoctial`, since both sets are different. Therefore:

``` python
def test_classical_to_modified_equinoctial_raises_singularity_error_if_inc_180_degrees():
```

another approach:

```python
def test_coe_to_mee_raises_singularity_error_orbit_equatorial_and_retrograde():
```

Reply:
Thanks for this!